godot-core/src/builtin/aabb.rs

```diff
@@ -53,16 +53,15 @@ impl Aabb {
 
     /// Set size based on desired end-point.
     ///
-    /// _Godot equivalent: `Aabb.size` property_
+    /// _Godot equivalent: `Aabb.size` property_
+    #[doc(alias = "size")]
     #[inline]
     pub fn set_end(&mut self, end: Vector3) {
         self.size = end - self.position
     }
```

Comment:
Size and end are different things. The existing "Godot equivalent" is wrong too. Can you remove both?